sysbuild/Kconfig.netcore

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) 2022 Nordic Semiconductor
+#
+# SPDX-License-Identifier: Apache-2.0
+
+config SUPPORT_NETCORE
+	bool
+	default y if $(BOARD) = "nrf5340dk_nrf5340_cpuapp"
```

Comment:
I assume out of tree users are going to need to have a custom module with a sysbuild Kconfig which adds new entries to this field?

Reply:
short term, yes.

but SoC v2 should allow us to make much better Kconfigs.
Important part for this PR is the sysbuild / pm integration.